docs/v2/features/repeatheader.md

```
@@ -0,0 +1,214 @@
+		panic(err)
+	}
+
+	doc.Save("catalog.pdf")
+}
+```
+
+## Output Structure
+
+When a page break occurs, the new page structure is:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)